common/helpers/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 name = "common-helpers"
 publish = false
 repository = "https://github.com/LibertyDSNP/frequency/"
-version = "0.0.0"
+version = "1.0.0+polkadot0.9.30"
```

Comment:
Do we know if there might be some problems down the road if we have versioning with `+polkadot0.9.30` in terms of problem parsing in some tools or etc.

Reply:
See #874 and its referenced Issues for the public version of that discussion